src/main/java/com/user/api/controller/UserController.java

```diff
@@ -0,0 +1,88 @@
+package com.user.api.controller;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.Reader;
+import java.sql.SQLException;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.user.api.dto.UserDto;
+import com.user.api.transformer.UserDtoTransformer;
+import com.user.db.entity.User;
+import com.user.service.UserService;
+
+@WebServlet("/user")
+public class UserController extends HttpServlet {
+
+	private static final long serialVersionUID = 1L;
+	private final Logger logger = LogManager.getLogger(UserController.class);
+	private final ObjectMapper mapper = new ObjectMapper();
+	private final UserService userService = new UserService();
+
+	protected void doGet(HttpServletRequest request, HttpServletResponse response)
+			throws ServletException, IOException {
+		User user = null;
+		String id = request.getParameter("id");
+		try {
+			user = userService.get(id);
+		} catch (SQLException e) {
+			logger.error(e.toString());
+		} catch (ClassNotFoundException e) {
+			logger.error(e.toString());
+		}
+		String userData = mapper.writeValueAsString(user);
+		response.setContentType("application/json");
+		response.getWriter().println(userData);
+	}
+
+	protected void doPost(HttpServletRequest request, HttpServletResponse response)
+			throws ServletException, IOException {
+		PrintWriter printWriter = response.getWriter();
+		response.setContentType("text/html");
+		Reader data = request.getReader();
+		User user = UserDtoTransformer.transform(mapper.readValue(data, UserDto.class));
+		try {
+			printWriter.println(userService.create(user));
+		} catch (SQLException e) {
+			logger.error(e.toString());
+		} catch (ClassNotFoundException e) {
+			logger.error(e.toString());
+		}
+	}
+
+	protected void doPut(HttpServletRequest request, HttpServletResponse response)
+			throws ServletException, IOException {
+		String id = request.getParameter("id");
+		Reader data = request.getReader();
+		UserDto userDto = new ObjectMapper().readValue(data, UserDto.class);
```

Comment:
Do not create a new `ObjectMapper` every time you need one. Consider creating a final class variable.